MonoGame.Framework/GameWindow.cs

```diff
@@ -186,25 +200,10 @@ public void EndScreenDeviceChange (string screenDeviceName)
 			EndScreenDeviceChange(screenDeviceName, ClientBounds.Width, ClientBounds.Height);
 		}
 
-	    /// <summary>
-	    /// Called when the window gains focus.
-	    /// </summary>
-		protected void OnActivated ()
```

Comment:
I know these are empty for the moment, but why remove them?  It is not the focus of the change to "clean-up", as those are separate tasks.  Please revert @Cn-mjt44 , in indeed this change is removing them and not just git "reordering things.